src/components/bluebutton.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import styles from './bluebutton.scss';
+
+const Button = ({ targetTo, label, ...props }) =>
+(
+  <div
```

Comment:
`<div>` is really generic and should not be used if a more descriptive element exists. In this case [`<button>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/button) is a better start.